### ISSUE TITLE
Unable to change temperature

### ISSUE THREAD

**Issue by netforceatg** [garbled1/balboa_homeassistan#2]
Just installed this through HACS, and everything seems to be working.

However, as soon as I try to change temperature(increase), it drops to 26degC, and I am unable to increase it. Any idea?

![image](https://user-images.githubusercontent.com/22895218/82662273-d937de00-9c2d-11ea-800d-412179bddc17.png)

**Comment by netforceatg** [comment]
![image](https://user-images.githubusercontent.com/22895218/82662331-f2408f00-9c2d-11ea-90ac-da53a70dbf77.png)

**Comment by garbled1** [comment]
Can you provide any logs from HA?  Maybe just grep for "balboa" in the logs?  Try adjusting temp and then grabbing logs, I want to see if it had an error communicating or something?

**Comment by netforceatg** [comment]
![image](https://user-images.githubusercontent.com/22895218/82726040-bebf3c80-9ce1-11ea-910e-c2d60c8fb210.png)

**Comment by netforceatg** [comment]
Seems like I get all the correct values(current states) from the component, but unable to change any of them. As you can see on the log entry from 10:37:21 I got this error when i tried to increase temp from 36.5 to 37.0 (but i actually jumped from 36.5 to 26 when I hit increse at HA Web UI)

**Comment by garbled1** [comment]
Oh, can you tell me what homeassistant says the current heat range is?  Easy way is to go to developer tools and display the attributes of the climate device.  I think I know what the problem is.. mine looks wrong too... might be an easy fix.

```
hvac_modes: heat, off
min_temp: 50
max_temp: 80
preset_modes: Ready, Ready in Rest, Rest
current_temperature: 104
temperature: 104
hvac_action: idle
preset_mode: Ready
friendly_name: Spa
supported_features: 17
```

I believe in balboa.py, the self.tmin and self.tmax are reversed.  I think the low temp pair needs to be first, not the high temp pair, I expect to see 80/104 there for mine.. If you see 10/26 there than that is the problem and I can fix quickly.

**Comment by netforceatg** [comment]
Yes, looks to be 10/26 as you say

![image](https://user-images.githubusercontent.com/22895218/82733335-6a34b500-9d13-11ea-9b13-cdc9351dfb03.png)

**Comment by garbled1** [comment]
If you update to master, it should work now.  Thanks for catching this.  Fix is in pybalboa 0.6

**Comment by netforceatg** [comment]
Just updated and seems to be working, thanks! :)

**Comment by stringy92** [comment]
I copied over the new manifest as the version available on HACS had Pybalboa set at 0.4, so i've updated this to 0.6 however i am getting the following error - 

2020-05-26 11:45:24 ERROR (MainThread) [pybalboa.balboa] Spa socket error: [Errno 32] Broken pipe
